pennylane/operation.py

```diff
@@ -2955,7 +2955,7 @@ def gen_is_multi_term_hamiltonian(obj):
     return isinstance(o, (qml.ops.Hamiltonian, qml.ops.LinearCombination)) and len(o.coeffs) > 1
 
 
-def enable_new_opmath():
+def enable_new_opmath(warn=True):
```

Comment:
Sorry to add to the burden here, but I think we'll now need to mention this briefly in the docstring.

Reply:
yes good point!

Reply:
updated ✅